CHANGELOG.md

```diff
@@ -1,8 +1,21 @@
 
+
```

Comment:
Remove added blank line?

Reply:
I didn't see this go, and not that the existing leading blank line should probably be there either.

Reply:
Fixed now.